server/sandbox_stop.go

```diff
@@ -24,10 +25,13 @@ func (s *Server) StopPodSandbox(ctx context.Context, req *pb.StopPodSandboxReque
 	if err != nil {
 		return nil, err
 	}
-
-	if err := s.netPlugin.TearDownPod(netnsPath, podNamespace, sb.id, podInfraContainer.Name()); err != nil {
-		return nil, fmt.Errorf("failed to destroy network for container %s in sandbox %s: %v",
-			podInfraContainer.Name(), sb.id, err)
+	if _, err := os.Stat(netnsPath); err == nil {
```

Comment:
@rhatdan your code is exiting the stop routine w/o cleaning the containers below the teardown (we should just skip the teardown, not entirely exit the stop request when netns path doesn't exit (or there's an error stat'ing)